doc/for-test-authors.rst

```diff
@@ -469,6 +488,13 @@ example::
 Most of the time, you will want to uses `The raises helper`_ instead.
 
 
+warning_message
+~~~~~~~~~~~~~~~
+
+Match against various attributes–category, message and filename to name a few—of
```

Comment:
Suggest using parenthesis instead of hyphens, as I think it makes it easier to read:

> Match against various attributes (category, message and filename to name a few) of...